style.css

```diff
@@ -0,0 +1,922 @@
+:root {
+  --font-family: 'Lato', sans-serif;
+}
+
+@media screen and (min-width: 768px) {
+  *,
+  *::after,
+  *::before {
+    list-style: none;
+    overflow-x: hidden;
+  }
+
+  body {
+    margin: 0;
+  }
+
+  .links {
+    background-color: #3e3e3e;
+    height: 30px;
+  }
+
+  nav ul {
+    display: flex;
+    font-family: var(--font-family);
+    font-size: 14px;
+    color: #272a31;
+    font-weight: 400;
+  }
+
+  .links ul {
+    display: flex;
+    font-size: 13px;
+    font-family: var(--font-family);
+    color: #fff;
+    margin-top: 0;
+    margin-bottom: -1rem;
+    padding-bottom: 0;
+    margin-right: 10rem;
+    position: relative;
+    float: right;
+  }
+
+  nav ul li {
+    margin: 0.5rem 1rem 0.5rem 1rem;
+  }
+
+  .links ul li {
+    margin: 0.3rem 1rem 0.2rem 1rem;
+  }
+
+  nav ul li:hover {
+    color: #ec5242;
+  }
+
+  .nav {
+    display: flex;
+    width: 100vw;
+    justify-content: space-around;
+    height: 75px;
+  }
+
+  footer strong {
+    font-size: 28px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    font-style: italic;
+  }
+
+  footer strong img {
+    height: 2.6rem;
+    margin-top: 0.5rem;
+  }
+
+  .nav strong a {
+    font-size: 28px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    font-style: italic;
+    color: #000;
+    text-decoration: none;
+  }
+
+  .feature img {
+    height: 10rem;
+    width: 9rem;
+  }
+
+  .red-img img {
+    height: 8rem;
+    width: 15rem;
+    margin: 0.5rem;
+  }
+
+  .nav strong img {
+    height: 2.6rem;
+    margin-top: 0.5rem;
+  }
+
+  .nav button {
+    color: #ec5242;
+    border: 3px solid #ec5242;
+    background-color: #fff;
+    height: 2.8rem;
+    font-family: var(--font-family);
+    font-size: 14px;
+    margin-top: 0.5rem;
+    padding: 0.5rem 1rem 0.5rem 1rem;
+  }
+
+  .click {
+    text-decoration: none;
+    color: #272a31;
+  }
+
+  .click:hover {
+    color: #ec5242;
+  }
+
+  .back {
+    background: url("images/1678984284631.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 740px;
+  }
+
+  .pad {
+    margin: 4rem 10rem 7rem 10rem;
+  }
+
+  .pad h2 {
+    color: #ec5242;
+    font-size: 33px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  }
+
+  .pad h1 {
+    font-size: 48px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    background-image: url("images/pexels-pixabay-267151.jpg");
+    background-size: contain;
+    background-position: top center;
+    -webkit-background-clip: text;
+    color: transparent;
+    margin-top: -1rem;
+  }
+
+  .box {
+    font-family: var(--font-family);
+    font-size: 25px;
+    width: 60vw;
+    padding: 2rem;
+    border: 3px solid #fff;
+    color: #272a31;
+  }
```

Comment:
- [ ] Again, You did a very great job ensuring that you have a theme description👏. We need to ensure that we stick to the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as much as possible. This will enable us to make the website more professional and user-friendly. Therefore I suggest the description box covers almost 50% of the screen size. In addition, ensure to use the same color used to match the original design. The colors recommended to use in this project include #272A31, #EC5242, and #D3D3D3☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/226077936-889aa558-b5b6-45ac-a145-6cc2117b700e.png)

Original design 😎 
![image](https://user-images.githubusercontent.com/99394374/226077949-7480c282-84c8-4d11-ada2-075c4ace3d1c.png)